lib/dl_api_lib/dl_api_lib/connector_availability/base.py

```diff
@@ -118,13 +121,15 @@ class Connector(ConnectorBase):
     conn_type: ConnectionType = attr.ib(
         converter=lambda v: ConnectionType(v) if not isinstance(v, ConnectionType) else v
     )
-    availability: ConnectorAvailability = attr.ib(default=ConnectorAvailability.free, converter=ConnectorAvailability)
+    visibility_mode: ConnectorAvailability = attr.ib(
+        default=ConnectorAvailability.free, converter=ConnectorAvailability
+    )
```

Comment:
Just wondering, why not leave `availability`?

Reply:
When we discussed it with Evgenyi, we agreed on such a name and I stuck to it